[2025-07-13T00:20:20.919+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flight_data_etl.load_to_database scheduled__2025-07-12T18:00:00+00:00 [queued]>
[2025-07-13T00:20:20.929+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flight_data_etl.load_to_database scheduled__2025-07-12T18:00:00+00:00 [queued]>
[2025-07-13T00:20:20.930+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-13T00:20:20.943+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_to_database> on 2025-07-12 18:00:00+00:00
[2025-07-13T00:20:20.950+0000] {standard_task_runner.py:60} INFO - Started process 12387 to run task
[2025-07-13T00:20:20.955+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'flight_data_etl', 'load_to_database', 'scheduled__2025-07-12T18:00:00+00:00', '--job-id', '175', '--raw', '--subdir', 'DAGS_FOLDER/flight_data_etl.py', '--cfg-path', '/tmp/tmp0hxby5fm']
[2025-07-13T00:20:20.958+0000] {standard_task_runner.py:88} INFO - Job 175: Subtask load_to_database
[2025-07-13T00:20:21.007+0000] {task_command.py:423} INFO - Running <TaskInstance: flight_data_etl.load_to_database scheduled__2025-07-12T18:00:00+00:00 [running]> on host f1b3ce86e5d1
[2025-07-13T00:20:21.291+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineer' AIRFLOW_CTX_DAG_ID='flight_data_etl' AIRFLOW_CTX_TASK_ID='load_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-07-12T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-12T18:00:00+00:00'
[2025-07-13T00:20:21.303+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-07-13T00:20:21.348+0000] {logging_mixin.py:188} INFO - Loaded 81 airport records
[2025-07-13T00:20:21.361+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-07-13T00:20:21.439+0000] {logging_mixin.py:188} INFO - Loaded 1000 scheduled flight records
[2025-07-13T00:20:21.470+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-07-13T00:20:21.920+0000] {logging_mixin.py:188} INFO - Loaded 7002 real-time flight records
[2025-07-13T00:20:21.922+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-13T00:20:21.931+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=flight_data_etl, task_id=load_to_database, execution_date=20250712T180000, start_date=20250713T002020, end_date=20250713T002021
[2025-07-13T00:20:21.982+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-13T00:20:21.995+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
